modules/tinymce/src/core/test/ts/browser/paste/PasteTest.ts

```diff
@@ -343,6 +364,22 @@ describe('browser.tinymce.core.paste.PasteTest', () => {
     assert.equal(PasteUtils.trimHtml('<span class="Apple-converted-space">\u00a0<\/span>'), ' ');
   });
 
+  it('TINY-9776: it should be possible to paste an image via mceInsertClipboardContent', async () => {
+    const editor = hook.editor();
+    editor.setContent('<p>text</p>');
+    TinySelections.setSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
+
+    TinyAssertions.assertContentPresence(editor, { img: 0 });
+
+    editor.execCommand('mceInsertClipboardContent', false, {
+      files: [ base64ToBlob(base64ImgSrc, 'image/gif', 'image.gif') ]
```

Comment:
Also feed it more than one image and also other mime types. It should not try to process those.

Reply:
added